package.json

```diff
@@ -58,7 +58,7 @@
     "ts-node": "^3.3.0",
     "tslint": "^4.5.1",
     "typedoc": "^0.7.1",
-    "typescript": "~2.4.0"
+    "typescript": "^2.7.0"
```

Comment:
This'll work, but if I were you I'd stay pinned a least to a minor version (`~2.7.0`). TS doesn't really follow semver, so bumps can do large breaking things, and this significantly increases the odds this project will randomly stop building in future.

Reply:
Whilst true about semver, I'm kinda of the opinion we should be 'keeping up with latest TS', and fixing issues as they come up. One of the reasons this languished so long was the work wasn't done to bring it up, and it just kept lapsing. :-/

I'll lock it, but we need to keep on top of version changes better than we have been.